Realm/RLMResults.mm

```diff
@@ -364,6 +364,32 @@ - (RLMResults *)sortedResultsUsingDescriptors:(NSArray<RLMSortDescriptor *> *)pr
     });
 }
 
+- (RLMResults *)distinctResultsUsingKeyPaths:(NSArray<NSString *> *)keyPaths {
+    if (keyPaths.count == 0) {
+        @throw RLMException(@"Must specify at least one keypath.");
```

Comment:
Is there any reason to throw in this case given that we don't when sorting?

Reply:
Nope